ast/ast.go

```diff
@@ -76,6 +76,14 @@ func (m *Model) FieldSlice() []string {
 	return out
 }
 
+func (m *Model) FieldSliceInCamelCase() []string {
+	out := []string{}
+	for _, v := range m.Fields {
+		out = append(out, v.NameInCamelCase())
+	}
+	return out
+}
+
```

Comment:
You really only need to lowerCase the first character. This code will do that without the library:
```go
out = append(out, strings.ToLower(v[:1])+v[1:])
```

Reply:
Okay. I picked the full camelCase to as a sledgehammer to fix the situation where `ID` would become `id`, instead of `iD`

Reply:
ah.. `id` is nicer that `iD`. @boourns, you cool with the library?

Reply:
Yep works for me

Reply:
<3 
This morning, I considered a way to handle those cases...and it basically becomes implementing what's in the library.